src/main/java/frc/robot/RobotContainer.java

```diff
@@ -138,10 +137,10 @@ private void configureButtonBindings()
     {
         _drive.setDefaultCommand(DriveCommands.joystickDrive(_drive, () -> -_joystick.getY(), () -> -_joystick.getX(), () -> -_joystick.getZ()));
 
-        _controller.y().whileTrue(NotepathCommands.startFeed(_notepath).andThen(Commands.idle(_notepath)).finallyDo(() -> _notepath.setOff()));
-        _controller.x().whileTrue(NotepathCommands.reverseFeed(_notepath).andThen(Commands.idle(_notepath)).finallyDo(() -> _notepath.setOff()));
-        _controller.a().whileTrue(IntakeCommands.startIntake(_intake).andThen(Commands.idle(_intake)).finallyDo(() -> _intake.setIntakeOff()));
-        _controller.b().whileTrue(IntakeCommands.reverseIntake(_intake).andThen(Commands.idle(_intake)).finallyDo(() -> _intake.setIntakeOff()));
+        _controller.y().onTrue(CompositeCommands.intakePickup(_intake, _notepath).andThen(CompositeCommands.stopIntaking(_intake, _notepath)));
+        _controller.x().onTrue(CompositeCommands.stopIntaking(_intake, _notepath));
+        _controller.a().onTrue(CompositeCommands.shooterPickup(_shooterBed, _shooterFlywheel, _notepath).andThen(IntakeCommands.stopIntake(_intake)));
+        _controller.b().onTrue(IntakeCommands.reverseIntake(_intake).until(() -> !_controller.b().getAsBoolean()).andThen(IntakeCommands.stopIntake(_intake)));
```

Comment:
I know you didn't change this line (it changed in a different PR), but we should match what's in main. This line has the potential to crash the code by modifying the button loop in some very specific circumstances we didn't completely figure out:

```suggestion
        _controller.b().whileTrue(IntakeCommands.reverseIntake(_intake).andThen(Commands.idle(_intake)).finallyDo(() -> _intake.setIntakeOff()));
```